src/services/github.ts

```diff
@@ -37,7 +37,7 @@ export default class GithubService {
 
       this.octokit = new RetryableThrottledOctokit({
         auth: token,
-        baseUrl: `https://${shepherdGitHubEnterpriseUrl}/api/v3`,
+        baseUrl: `https://${shepherdGitHubEnterpriseUrl}`,
```

Comment:
/api/v3 is required for GITHUB enterprise server apis, for the cloud the api/v3 is removed from the baseurl
https://docs.github.com/en/enterprise-cloud@latest/rest/quickstart?apiVersion=2022-11-28

Reply:
Tell me if I am missing something, but I do not see any `/api/v3` references in the docs that you provided. In the same way, I do not see a reference to `/api/v3` [anywhere in the enterprise docs](https://docs.github.com/en/enterprise-cloud@latest/search?query=api%2Fv3). In all of the examples in the quickstart you reference, GH uses examples like:
```
curl --request GET \
--url "https://api.github.com/repos/octocat/Spoon-Knife/issues" \
--header "Accept: application/vnd.github+json" \
--header "Authorization: Bearer YOUR-TOKEN"
```
Where the base URL does not have the `/api/v3` suffix.